README.md

```diff
@@ -106,3 +106,16 @@ We're working hard on the new redesigned release in the new separate `main-next`
 • More printed schedules and forms (like S-12, S-13, S-140 variations, and more) <br>
 
 We're excited about adding these and many more new features! Of course, it takes time to develop them... If you know programming and want to help, we'd really appreciate it! Together we can create a better app for the brotherhood across the whole globe!
+
+## Useful links
+
+To learn more about Organized and how to use it, check out the following useful links:
+
+| Link                                                         | Description                                                   |
+|-------------------------------------------------------------------------------|---------------------------------------------------------------|
+| [Organized app](organized-app.com)                                           | Open and use the Organized app.           |
```

Comment:
@ux-git: would it possible to fix some of the links to begin with `https://`? Without this, the links won’t work. Thank you.